### PR TITLE
[WIP] Alternate observability pipeline image and architecture

### PR DESCRIPTION
**Pull request by hillaryfraley**
## Description
Alternative vision of the observability pipeline and associated architecture based on ideas from the developer advocates.

## Motivation and Context
Docs IA project

### CODE REVIEW

File: content/sensu-go/6.0/observability-pipeline/_index.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+product: "Sensu Go"
+version: "6.0"
+weight: 10
+layout: "single"
+toc: false
+menu:
+  sensu-go-6.0:
+    identifier: observability-pipeline
+---
+
```
Comment (hillaryfraley) [inline content/sensu-go/6.0/observability-pipeline/_index.md:14]: @cwjohnston here is the observability pipeline text that you mentioned you had some thoughts about.

File: content/sensu-go/6.0/observability-pipeline/_index.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+Sensu's observability pipeline is a flexible, automated tool that gives you visibility into every part of your organization's infrastructure.
+
+The Sensu [agent][1] is a lightweight client that runs on your [entities][2], which are the infrastructure components you want to observe.
```
Comment (cwjohnston) [inline content/sensu-go/6.0/observability-pipeline/_index.md]:
```suggestion
The Sensu [agent][1] is a lightweight process that runs on the infrastructure components you want to observe.
Each agent is represented in Sensu as an [entity][2].
```

If we feel it's important to address infra that we can't run an agent on, we might consider adding additional text like "Entities can also represent infrastructure which is observed by an agent acting as a proxy for entities which cannot run their own agent process." to explain proxy entities. I don't know if thats appropriate for this section.

File: content/sensu-go/6.0/observability-pipeline/_index.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+The Sensu [backend][1] schedules [checks][3] for agents to run on your entities based on the agent subscriptions you specify.
```
Comment (cwjohnston) [inline content/sensu-go/6.0/observability-pipeline/_index.md]:
```suggestion
The Sensu [backend][1] schedules [checks][3] for agents to run on your infrastructure.
Agents receive check execution requests based on the agent subscriptions you specify.
```

File: content/sensu-go/6.0/observability-pipeline/_index.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+The agent runs these checks on your entities to gather observation data about them.
```
Comment (cwjohnston) [inline content/sensu-go/6.0/observability-pipeline/_index.md]:
```suggestion
The agent runs these checks on your infrastructure to gather observation data about your networking, compute resources, applications, and more .
```

**Review by cwjohnston — CHANGES_REQUESTED** [review]
Suggested some changes to observability-pipeline/_index.md, mostly wording that might better help evaluators understand what agents do.

**Comment by hillaryfraley** [comment]
Closed in favor of the final PR: https://github.com/sensu/sensu-docs/pull/2709

**Pull request closed without merging**